tasks/build-link

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env node
```

Comment:
And this is how we're copying only relevant files, and using esbuild to build the framework. There must be some config missing in esbuild